styleguide.css

```diff
@@ -308,10 +315,10 @@ form#signin {
     content: '\ea10';
     font-style: normal;
```

Comment:
is this required? `normal` is the default for a label, right?

Reply:
ok, not required then